src/modules/formBuilder/components/FormBuilderPalette.tsx

```diff
@@ -135,8 +135,13 @@ const FormBuilderPalette: React.FC<FormBuilderPaletteType> = ({
     >
       <Stack gap={1}>
         {Object.entries(FORM_ITEM_PALETTE)
-          // TODO - Object is a special case, remove it from the list for now until we support it
-          .filter(([key]) => key !== ItemType.Object)
+          // TODO - Image, File, and Object are special cases. Remove from the list for now until we add support
```

Comment:
Don't think this needs to be marked as a TODO since there is no immediate action, maybe add a reference to issue 6401 instead? https://github.com/open-path/Green-River/issues/6401